app/views/govuk_publishing_components/components/_textarea.html.erb

```diff
@@ -17,6 +17,8 @@
   hint_id = "hint-#{SecureRandom.hex(4)}"
   has_error ||= error_message || error_items.any?
   error_id = "error-#{SecureRandom.hex(4)}"
+  text_direction_rtl ||= false
+  text_direction_label_ltr ||= false
```

Comment:
General suggestion - could the variable names used be clearer? Maybe use a general format like `text_direction_[element]_[direction]` e.g. `text_direction_input_right ||= false`. Later variables like `dir_textarea` could also use this convention for consistency. Distinguishing between `ltr` and `rtl` compared to `left` and `right` at a glance is harder.